Userland/Utilities/lscpu.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022, Undefine <undefine@undefine.pl>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Format.h>
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+
+static void parse_processor(JsonValue const& value)
```

Comment:
The name is a bit weak - something like print_cpu_info seems better.
Also, pass an object instead of a value, then we don't have to as_object() below